pkgs/by-name/jj/jjui/package.nix

```diff
@@ -6,30 +6,28 @@
 }:
 buildGoModule rec {
   pname = "jjui";
-  version = "0.4";
+  version = "0.5";
 
   src = fetchFromGitHub {
     owner = "idursun";
     repo = "jjui";
     tag = "v${version}";
-    hash = "sha256-PAquHh3VVAhc4Tw1XkyKmwFbpLVkDRCMT+FGtqiydCA=";
+    hash = "sha256-+1KVKevY7aWkAbbHQi06whh3keibdSVEykYDItSMi4I=";
   };
 
   vendorHash = "sha256-MxTwe0S2wvkIy8VJl1p8utTX98zfcwpNgCdnpFAMxO0=";
 
-  postFixup = ''
-    mv $out/bin/cmd $out/bin/jjui
-  '';
-
   passthru.updateScript = nix-update-script { };
 
   meta = {
     description = "A TUI for Jujutsu VCS";
     homepage = "https://github.com/idursun/jjui";
+    changelog = "https://github.com/<repo>/<package>/releases/tag/v${version}";
     license = lib.licenses.mit;
     maintainers = with lib.maintainers; [
       adda
     ];
+    platforms = lib.platforms.all;
```

Comment:
This is automatically set by buildGoModule.

Further, Go is not implemented in all archs anq OSes yet.

```suggestion
```

Reply:
Thanks for the heads-up. Good to know. Removed again.